istioctl/pkg/internaldebug/internal-debug.go

```diff
@@ -36,10 +36,10 @@ func HandlerForRetrieveDebugList(kubeClient kube.CLIClient,
 	centralOpts clioptions.CentralControlPlaneOptions,
 	writer io.Writer,
 	istioNamespace string,
-) (map[string]*discovery.DiscoveryResponse, error) {
+) (map[string]*discovery.DeltaDiscoveryResponse, error) {
```

Comment:
I do not get the intention here, for debug it is a one time behavior. I cannot see the benefit to change to delta xds

Reply:
The intention was to avoid using the stow client, and I think we will eventually need to switch to delta xDS, including the istioctl client usage as well. Currently, there's no benefit for status/debug gen, as they only retrieve resources once. I think it is simpler to make the switch, and right now the only usage of ADSC besides tests are istioctl commands, so I included the change in the same PR.

> I donot think we should transit to using the delta xds in istioctl immediately

Do you think we need to switch tests first and then other things?

Reply:
To be clear, i donot think we will remove supporting Stow. Delta is an option with feature flag. 

And implementing adsc delta is not equal to transiting to delta xds.

Reply:
I see, thanks for the clarification! I'll remove the command changes, and if we need them later, we can do a cherry-pick. This should make the PR cleaner.